dmt/asn2aadlPlus.py

```diff
@@ -144,8 +143,8 @@ def calculateForNativeAndASN1SCC(absASN1SCCpath, autosrc, names, inputFiles):
         pipe = Popen("find-supported-compilers", stdout=PIPE).stdout
         g_platformCompilers = pipe.read().splitlines()
     except OSError as err:
-        print('Not running in a TASTE environment: {}\nUsing GCC only for computing sizeofs'.format(str(err)))
-        g_platformCompilers = ['gcc']
+        print('Not running in a TASTE Environment: {}\nUsing GCC only for computing sizeofs'.format(str(err)))
+        g_platformCompilers = ['gcc'.encode()]
```

Comment:
In python 3 all literal strings are unicode by default. The script from line 144 reports:
```
pipe = Popen("find-supported-compilers", stdout=PIPE).stdout
g_platformCompilers = pipe.read().splitlines()
```
where `g_platformCompilers` is a list of byte literals (introduced from python3). 
```
g_platformCompilers = ['gcc'.encode()]
```
This line of code defines the same list of bytes literal if the the script is run outside the TASTE environment. In this way the function `decode()` called at line 154 does not arise errors like 
```
AttributeError: 'cc' object has no attribute 'decode'
```
in the case the script is run outside the TASTE environment.